pkg/windows/windows.go

```diff
@@ -489,6 +493,7 @@ func (vm *windows) createDirectories() error {
 
 // removeDirectories removes all directories created as part of the configuration process
 func (vm *windows) removeDirectories() error {
+	vm.log.Info("deleting directories")
```

Comment:
s/deleting/removing

for consistency with the func signature and [error](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/windows/windows.go#L494):
```
return errors.Wrapf(err, "unable to remove directory %s", dir)
```